conda_environments/testing_py35.yml

```diff
@@ -1,5 +1,6 @@
 name: pycudf_testing_py35
 channels:
+- nvidia
 - numba
 - conda-forge
 - rapidsai
```

Comment:
Suggest moving rapidsai above conda-forge in case someone starts repackaging rapids libraries in conda-forge.

Reply:
Why didn't this comment get incorporated in the final PR?

Reply:
It's moreso nitpicking and can be addressed in the future.